bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx

```diff
@@ -90,9 +91,14 @@ class UserListContent extends Component {
 
   componentDidUpdate() {
     this.checkDropdownDirection();
+    if (this.navbarDefaultClass === null && this.props.navbarRef !== null) {
+      this.navbarDefaultClass = this.props.navbarRef.className;
+    }
   }
 
   onActionsShow() {
+    this.props.navbarRef.className = `${this.navbarDefaultClass}Hide`;
```

Comment:
You aren't doing this the "React Way". In React, you should ideally not be manipulating anything via the DOM.

Reply:
How about Manipulating virtual dom?

Reply:
I wonder that wrapping navbar by ref and sending it to the children components and manipulate the ref at the children components is wrong way??

Reply:
you should manipulate any DOM property by yourself, refs are DOMNodes and here you are manipulating the property className without the React knowledge.